epoxy-processor/src/main/java/com/airbnb/epoxy/PoetExtensions.kt

```diff
@@ -54,18 +55,18 @@ fun JavaClassName.toKPoet(): KotlinClassName {
 
 /** Some classes, like List or Byte have the same class name but a different package for their kotlin equivalent. */
 private fun JavaClassName.getPackageNameInKotlin(): String {
-    if (packageName() in listOf(javaUtilPkg, javaLangPkg) && simpleNames().size == 1) {
+    if (packageName() in listOf(javaUtilPkg, javaLangPkg, kotlinJvmFunction) && simpleNames().size == 1) {
 
-        val transformedPkg = if (isBoxedPrimitive) {
-            kotlinPkg
-        } else {
-            when (simpleName()) {
+        val transformedPkg = when {
+            isBoxedPrimitive -> kotlinPkg
+            isLambda(this) -> kotlinPkg
+            else -> when (simpleName()) {
                 "Collection",
                 "List",
                 "Map",
                 "Set",
                 "Iterable" -> kotlinCollectionsPkg
-                "String" -> kotlinPkg
+                "String", "Function1" -> kotlinPkg
```

Comment:
is it necessary to check `Function1` since the `isLambda` check is already done?

Reply:
Oops that's my bad. Leftover code from when i was experimenting.